### PR TITLE
fix(scss) add ellipsis on chip in lu-select to prevent multiline

### PR DESCRIPTION
**Pull request by BertrandPodevin**
Fix https://github.com/LuccaSA/lucca-front/issues/1237

Add ellipsis on chips inside lu-select

chips

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1248

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1248

File: packages/ng/libraries/root/src/style/definitions/select/_select-input.scss [hunk 1]
```diff
@@ -136,10 +136,15 @@
...
 				height: _theme("sizes.small.line-height");
 				line-height: _theme("sizes.small.line-height", true);
 				line-height: _theme("sizes.small.line-height");
+				text-overflow: ellipsis;
+				overflow: hidden;
+				vertical-align: top;
+				white-space: nowrap;
 			}
 		}
 	}
```
[the remark this answers is not in the document]
Comment (lucienbertin) [reply packages/ng/libraries/root/src/style/definitions/select/_select-input.scss]: english !!!!

[the remark this answers is not in the document]
Comment (lucienbertin) [reply packages/ng/libraries/root/src/style/definitions/select/_select-input.scss]: this code is placed on the chip present in the mod-small uniquelly, it must it go up no ?

[the remark this answers is not in the document]
Comment (AlexiaAlarza) [reply packages/ng/libraries/root/src/style/definitions/select/_select-input.scss]: my bad !

**Review by AlexiaAlarza — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1248

**Pull request merged**